test/core/Do.Test.Architecture/Architecture/Application/RunningAnApplication.cs

```diff
@@ -171,7 +172,7 @@ public void Only_one_phase_can_have_earliest_and_latest_priorities_at_the_same_t
 
         var app = GiveMe.AnApplication(layer: layer);
 
-        Assert.That(() => app.Run(), Throws.TypeOf<OverlappingPhaseException>());
+        Should.Throw< OverlappingPhaseException>(() => app.Run());
```

Comment:
wrong usage, `action.ShouldThrow<OverlappingPhaseException>()`

please refer to our assertion guid :point_right: https://github.com/mouseless/learn-dotnet/blob/main/unit-testing/test/UnitTesting/Assertions.cs#L26